ErpFetch.js

```diff
@@ -75,7 +75,14 @@ export class ErpFetch {
         } else if (responseType === 'arrayBuffer') {
           return response.arrayBuffer()
         } else if (responseType === 'json') {
-          return response.json()
+          // check if response is empty (204) and return null
+          const contentType = response.headers.get('content-type');
+          if (response.status !== 204 && contentType === 'application/json') {
+            return response.json();
+          } else {
+            return {};
+          }
+
```

Comment:
I would return `null` on 204.

And I would not check for the content type, as this may create backward compatibility issues?

Reply:
@sbidoul you're right. fixed.